.nuget/create_nuget.py

```diff
@@ -36,7 +40,7 @@
         <dependencies> </dependencies>
     </metadata>
         <files>
-        <file src="runtimes\**" target="runtimes"/>
+        <file src="lib\**" target="lib"/>
```

Comment:
@codemzs I am not sure about this. Is this correct?

Reply:
I'm not sure either. Can you please send me this nuget so that I can try locally? 

---
In reply to: [207795534](https://github.com/Microsoft/LightGBM/pull/1569#discussion_r207795534) [](ancestors = 207795534)

Reply:
[Nuget.zip](https://github.com/Microsoft/LightGBM/files/2264737/Nuget.zip)

Reply:
you also can download it from vsts bulid: https://lightgbm.visualstudio.com/LightGBM-Build/LightGBM-Build%20Team/_build